modules/manage/pages/iceberg/access-iceberg-topics.adoc

```diff
@@ -0,0 +1,103 @@
+
+To integrate a REST catalog, set the following cluster configuration properties:
+
+* config_ref:iceberg_catalog_type,true,properties/cluster-properties[`iceberg_catalog_type`]: `rest`
+* config_ref:iceberg_rest_catalog_endpoint,true,properties/cluster-properties[`iceberg_rest_catalog_endpoint`]: The endpoint URL for your Iceberg catalog, which you either manage directly, or is managed by an external catalog service.
+* config_ref:iceberg_rest_catalog_client_id,true,properties/cluster-properties[`iceberg_rest_catalog_client_id`]: The ID to connect to the REST catalog.
+* config_ref:iceberg_rest_catalog_client_secret,true,properties/cluster-properties[`iceberg_rest_catalog_client_secret`]: The secret data to connect to the REST catalog.
+
+For REST catalogs that use self-signed certificates, also configure these properties:
+
+* config_ref:iceberg_rest_catalog_trust_file,true,properties/cluster-properties[`iceberg_rest_catalog_trust_file`]: The path to a file containing a certificate chain to trust for the REST catalog.
+* config_ref:iceberg_rest_catalog_crl_file,true,properties/cluster-properties[`iceberg_rest_catalog_crl_file`]: The path to the certificate revocation list for the specified trust file.
+
+See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
+
+=== Example REST catalog configuration
+
+For example, if you have Redpanda cluster configuration properties set to connect to a REST catalog:
+
+[,yaml]
+----
+iceberg_catalog_type: rest 
+iceberg_rest_catalog_endpoint: http://catalog-service:8181 
+iceberg_rest_catalog_client_id: <rest-connection-user>
+iceberg_rest_catalog_client_secret: <rest-connection-password>
+----
+
+And you use Apache Spark as a processing engine, configured to use a catalog named `streaming`:
+
+[,spark]
+----
+spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.streaming.type = rest
+spark.sql.catalog.streaming.uri = http://catalog-service:8181
+----
+
+Using Spark SQL, you can query the Iceberg table directly by specifying the catalog name:
+
+[,sql]
+----
+SELECT * FROM streaming.redpanda.<topic-name>;
+----
+
+Spark can use the REST catalog to automatically discover the topic's Iceberg table. Redpanda puts the Iceberg table into a namespace called `redpanda`, creating the namespace if necessary.
+
+Depending on your processing engine, you may need to also create a table in the engine to point the data lakehouse at the table in the catalog. For an example, see xref:manage:iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc[].
```

Comment:
```suggestion
Depending on your processing engine, you may need to also create a table in the engine to point the data lakehouse at the table in the catalog. For an example, see xref:manage:iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc[].
```
This isn't clear to me. Is it missing a word?

Reply:
Updated